content/en/docs/concepts/containers/_index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Containers"
 weight: 40
+description: Technology for packaging the (compiled) code for an application along with the dependencies it needs at run time
```

Comment:
nits:
Sentence does not end with period.
`Technology for packaging the ...` . How is this page different from the general definition of containers, such as,
https://docs.microsoft.com/en-us/dotnet/architecture/microservices/container-docker-introduction/

Reply:
This text is borrowed from https://kubernetes.io/docs/concepts/containers/overview/
I was going to leave wording changes for a follow-up PR.

I'm happy to use different wording here; do you have something in mind?